conf/genomes.config

```diff
@@ -25,34 +25,38 @@ params {
       vepCacheVersion  = "95"
     }
     'GRCh38' {
-      acLoci           = "${params.genomes_base}/1000G_phase3_GRCh38_maf0.3.loci"
-      acLociGC         = "${params.genomes_base}/1000G_phase3_GRCh38_maf0.3.loci.gc"
-      bwaIndex         = "${params.genomes_base}/Homo_sapiens_assembly38.fasta.64.{alt,amb,ann,bwt,pac,sa}"
-      dbsnp            = "${params.genomes_base}/dbsnp_146.hg38.vcf.gz"
-      dbsnpIndex       = "${params.genomes_base}/dbsnp_146.hg38.vcf.gz.tbi"
-      genomeDict       = "${params.genomes_base}/Homo_sapiens_assembly38.dict"
-      genomeFile       = "${params.genomes_base}/Homo_sapiens_assembly38.fasta"
-      genomeIndex      = "${params.genomes_base}/Homo_sapiens_assembly38.fasta.fai"
-      intervals        = "${params.genomes_base}/wgs_calling_regions.hg38.bed"
-      knownIndels      = "${params.genomes_base}/{Mills_and_1000G_gold_standard.indels.hg38,beta/Homo_sapiens_assembly38.known_indels}.vcf.gz"
-      knownIndelsIndex = "${params.genomes_base}/{Mills_and_1000G_gold_standard.indels.hg38,beta/Homo_sapiens_assembly38.known_indels}.vcf.gz.tbi"
-      snpeffDb         = "GRCh38.86"
-      vepCacheVersion  = "95"
+      acLoci                = "${params.genomes_base}/1000G_phase3_GRCh38_maf0.3.loci"
+      acLociGC              = "${params.genomes_base}/1000G_phase3_GRCh38_maf0.3.loci.gc"
+      bwaIndex              = "${params.genomes_base}/Homo_sapiens_assembly38.fasta.64.{alt,amb,ann,bwt,pac,sa}"
+      dbsnp                 = "${params.genomes_base}/dbsnp_146.hg38.vcf.gz"
+      dbsnpIndex            = "${params.genomes_base}/dbsnp_146.hg38.vcf.gz.tbi"
+      germlineResource      = "${params.genomes_base}/GCF_000001405.38.AUTOSOMESXY.COMMON.BIALLELIC.SNPs.with.AF.vcf.gz"
```

Comment:
Can you describe this references in `docs/usage.md`, `docs/reference.md` and in the `helpMessage` in `main.nf`